abn/watcher/informer.go

```diff
@@ -55,13 +82,12 @@ func NewIter8Watcher(configFile string) *Iter8Watcher {
 
 		// create informer for each resource in namespace
 		for gvr, validNames := range byResource {
+			gvr := gvr
```

Comment:
Please also add `validNames := validNames`. Note that both the loop variables are used in a closure.